webserver/external/messybrainz.py

```diff
@@ -1,35 +1,33 @@
 import requests
-import ujson
+import config
```

Comment:
In webserver you shouldn't reference config file directly. Use `current_app.config`.